atlp-rwanda/eagles-ec-fe
# #187419126 A seller should be able to update & delete a product item from their collection

#### What does this PR do?

This PR Adds product update and deletion of products by sellers

#### Description of Task to be completed?

- A seller should be able to view a collection of their products
- When a seller clicks on edit button on a product they should be redirected to a form with the product information where they can edit and save changes
- When a seller clicks on delete icon of a product they should be asked for confirmation to delete or cancel the action
- Test the feature

#### How should this be manually tested?
1. Clone the repository.
2. Checkout to the branch `ft-product-update-delete`.
3. Run `npm install` to install dependencies.
4. Run `npm run dev` to start the development server and login as a seller
5. navigate to `/dashboard/products`
6. Test if everything is working as expected

#### What are the relevant pivotal tracker stories?

Delivers #187419126

### Screenshots

Design 

<img width="754" alt="Screenshot 2024-07-02 at 5 49 02 PM" src="https://github.com/atlp-rwanda/eagles-ec-fe/assets/97763818/27a948bf-d948-4c06-8058-52275aa78b0b">

Implementation

<img width="1680" alt="Screenshot 2024-07-02 at 5 45 32 PM" src="https://github.com/atlp-rwanda/eagles-ec-fe/assets/97763818/2250e0a4-7718-40ef-b8fd-8bac7ad34281">

<img width="1680" alt="Screenshot 2024-07-02 at 5 45 41 PM" src="https://github.com/atlp-rwanda/eagles-ec-fe/assets/97763818/994a4634-9011-45b3-a378-42b31f59b800">

<img width="1680" alt="Screenshot 2024-07-02 at 5 45 56 PM" src="https://github.com/atlp-rwanda/eagles-ec-fe/assets/97763818/dd8760cb-77d7-4e8e-a503-537158aec4c1">
